cocoa-foundation/src/foundation.rs

```diff
@@ -240,7 +240,8 @@ impl NSProcessInfo for id {
     }
 
     unsafe fn isOperatingSystemAtLeastVersion(self, version: NSOperatingSystemVersion) -> bool {
-        msg_send![self, isOperatingSystemAtLeastVersion: version]
+        let res: BOOL = msg_send![self, isOperatingSystemAtLeastVersion: version];
+        res != NO
```

Comment:
Yup. Doesn't matter on Aarch64, since there it literally is `bool`, but on x86_64, `isOperatingSystemAtLeastVersion:` could _technically_ be returning e.g. `2`, and we'd have to treat that as `true`.